Packaging/Sources/5_build_libs-base.sh

```diff
@@ -13,8 +13,9 @@ if [ ${OS_ID} = "debian" ] || [ ${OS_ID} = "ubuntu" ]; then
 	sudo apt-get install -y ${GNUSTEP_BASE_DEPS} || exit 1
 else
 	${ECHO} "RedHat-based Linux distribution: calling 'yum -y install'."
-	SPEC_FILE=${PROJECT_DIR}/Libraries/gnustep/nextspace-gnustep.spec
-	DEPS=`rpmspec -q --buildrequires ${SPEC_FILE} | grep -v libobjc2 | awk -c '{print $1}'`
+	SPEC_FILE1=${PROJECT_DIR}/Libraries/gnustep/nextspace-gnustep.spec
+	SPEC_FILE2=${PROJECT_DIR}/Libraries/libcorefoundation/libcorefoundation.spec
```

Comment:
Why here? CoreFoundation is a separate library. GNUstep doesn't need it.

Reply:
This is the build step that failed for me. You wrote in [issue 481](https://github.com/trunkmaster/nextspace/issues/481#issuecomment-3059214347) that I need to run all 1-4 scripts. I assumed from [issue 446](https://github.com/trunkmaster/nextspace/issues/446#issuecomment-2514338118) that only 3-7 were necessary. Maybe there's a misunderstanding here.

Reply:
Removed this.